### DISCUSSION TITLE
Why is the network metric in the metrics section yellow?

### DISCUSSION THREAD

**Discussion by HAPPY-FAMILY** [minio/minio#20532] [Q&A] [answered]
![1](https://github.com/user-attachments/assets/179c4525-9130-4e85-b6ea-94467020ac61)
![2](https://github.com/user-attachments/assets/459387f2-e351-4e57-a6ad-385781bc900d)
![3](https://github.com/user-attachments/assets/8d9806ec-fca4-4c72-9019-a6716bba9e6a)
Although it is currently working normally, I am not sure if there are any potential issues. Could you please advise on what steps I need to take to resolve this problem?

**Comment by harshavardhana** [comment, marked as answer]
The setup is wrong, you are mixing two single node ECs into two pools. There is really no HA here.
